### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by taniyaabbineni14** [community/community#106885] [Other Feature Feedback, Questions, & Ideas]
### Hi everyone,

Hi Everyone 
I am New to Github Education

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @taniyaabbineni14  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be closed and locked to keep our community nice and tidy 🧹.
